src/interact.js

```diff
@@ -129,6 +130,12 @@ class Interact {
                 return false;
             }
 
+
+            if (game.player.hero === null) {
+                game.input("You do not have a hero.\n".red);
```

Comment:
Maybe have this throw an error instead?

Reply:
This is never supposed to happen, and is not supported, but do we NEED to throw an error if all files respect the typescript compiler and null-check before using `game.player.hero`?

Reply:
I won't remove this, but add this to #259